src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_users.erl

```diff
@@ -136,7 +136,8 @@ verbose_user(#chef_user{username = UserName, email = EMail, serialized_object =
     EJ = chef_json:decode(SerializedObject),
     {UserName, {[ {<<"email">>, EMail},
                   {<<"first_name">>, ej:get({<<"first_name">>}, EJ, "")},
-                  {<<"last_name">>, ej:get({<<"last_name">>}, EJ, "")} ]} }.
+                  {<<"last_name">>, ej:get({<<"last_name">>}, EJ, "")} ]},
+                  {<<"display_name">>, ej:get({<<"display_name">>}, EJ, "")} }.
```

Comment:
You'll need to modify the query[1] to include the new field as well, because `list_users_verbose` query called from `verbose_users_as_ejson` does not return `display_name`

[1] https://github.com/chef/chef-server/blob/master/src/oc_erchef/apps/chef_db/priv/pgsql_statements.config#L574

Reply:
@marcparadise This is pulling display_name from the serialized_object so I don't think a query change is required.  The display_name isn't otherwise part of the schema so the serialized_object is the source of truth for this field.

Reply:
@marcparadise I have updated the test case.